pkg/controller/injector/k8s_binding_injector.go

```diff
@@ -69,18 +68,24 @@ func (b *k8sBindingInjector) Inject(binding *servicecatalog.Binding, cred *broke
 }
 
 func (b *k8sBindingInjector) Uninject(binding *servicecatalog.Binding) error {
-	return fmt.Errorf("Not implemented")
+	// TODO: don't hardcode this namespace. https://github.com/kubernetes-incubator/service-catalog/issues/162
```

Comment:
That's also consistent with the overall plan for bindings we decided upon at the previous face-to-face